openquake/hazardlib/shakemap/parsers.py

```diff
@@ -1366,7 +1366,7 @@ def get_shakemap_versions(usgs_id, user=User(), monitor=performance.Monitor()):
     return shakemap_versions, usgs_preferred_version, err
 
 
-def _convert_rupture_file(rupture_file, usgs_id, user):
+def _convert_rupture_file(dic, rupture_file, usgs_id, user):
     rup = None
```

Comment:
I would like a better name than `dic`, making clear which parameters it contains